lms/static/js/spec/student_account/login_spec.js

```diff
@@ -212,6 +212,21 @@
                     expect($('.forgot-password')).toBeVisible();
                 });
 
+                it('displays password reset success message after password reset request', function() {
+                    createLoginView(this);
+
+                // Verify that the success message is not visible
+                    expect(view.$formFeedback.find('.submission-success').length).toEqual(0);
+
+                /* After a successful password reset request, the resetModel will trigger a 'sync'
+                 * event, which lets the LoginView know to render the password reset success message.
+                 */
+                    view.resetModel.trigger('sync');
+
+                // Verify that the success message is not visible
```

Comment:
And yet the below verifies that the success message *is* visible. Please update the comment.

Less important: align the comments with the lines they refer to (use the same indentation), though I see that a lot of this code seems to use the weird offset comments.